packages/aws-cdk-lib/aws-cloudwatch/README.md

```diff
@@ -496,6 +496,18 @@ dashboard.addWidgets(new cloudwatch.GraphWidget({
 }));
 ```
 
+When the `view` property is set to `GraphWidgetView.PIE`, the `labels` property can be used to toggle whether the magnitude of a pie segment is shown on the graph as a percentage of the whole.
```

Comment:
Again, I think `visible: false` can be used for all other widgets types, pie charts just happen to have a different default value?